api/src/test/java/com/microsoft/gctoolkit/time/DateTimeStampTest.java

```diff
@@ -334,134 +334,107 @@ public void timeSpanInMinutes() {
         DateTimeStamp greater = new DateTimeStamp("2018-04-04T10:10:00.587-0100");
         assertEquals(-1, smaller.compareTo(greater));
     }
 
     @Test
-    public void testCompareGreaterThan() {
+    void testCompareGreaterThan() {
         DateTimeStamp smaller = new DateTimeStamp("2018-04-04T09:10:00.586-0100");
         DateTimeStamp greater = new DateTimeStamp("2018-04-04T10:10:00.587-0100");
         assertEquals(1, greater.compareTo(smaller));
     }
 
     @Test
-    public void testCompareEquals() {
+    void testCompareEquals() {
         DateTimeStamp object1 = new DateTimeStamp("2018-04-04T10:10:00.586-0100");
         DateTimeStamp object2 = new DateTimeStamp("2018-04-04T10:10:00.586-0100");
         assertEquals(0, object1.compareTo(object2));
     }
 
     @Test
-    public void testCompareEqualsTimeStamp() {
+    void testCompareEqualsTimeStamp() {
         DateTimeStamp object1 = new DateTimeStamp("2018-04-04T10:10:00.586-0100", 123);
         DateTimeStamp object2 = new DateTimeStamp("2018-04-04T10:10:00.586-0100", 123);
         assertEquals(0, object2.compareTo(object1));
     }
 
     @Test
-    public void testCompareGreaterTimeStamp() {
+    void testCompareGreaterTimeStamp() {
         DateTimeStamp smaller = new DateTimeStamp("2018-04-04T10:10:00.586-0100", 122);
         DateTimeStamp greater = new DateTimeStamp("2018-04-04T10:10:00.586-0100", 123);
         assertEquals(1, greater.compareTo(smaller));
     }
 
     @Test
-    public void testCompareSmallerTimeStamp() {
+    void testCompareSmallerTimeStamp() {
         DateTimeStamp smaller = new DateTimeStamp("2018-04-04T10:10:00.586-0100", 123);
         DateTimeStamp greater = new DateTimeStamp("2018-04-04T10:10:00.586-0100", 124);
         assertEquals(-1, smaller.compareTo(greater));
     }
 
     @Test
-    public void testCompareNullValue() {
+    void testCompareNullValue() {
         DateTimeStamp smaller = new DateTimeStamp("2018-04-04T09:10:00.586-0100");
         assertEquals(-1, smaller.compareTo(null));
     }
 
     @Test
-    public void testCompareEqualsTimeStampWithoutDateTime() {
+    void testCompareEqualsTimeStampWithoutDateTime() {
         DateTimeStamp object1 = new DateTimeStamp(123);
         DateTimeStamp object2 = new DateTimeStamp(123);
         assertEquals(0, object2.compareTo(object1));
     }
 
     @Test
-    public void testCompareGreaterTimeStampWithoutDateTime() {
+    void testCompareGreaterTimeStampWithoutDateTime() {
         DateTimeStamp smaller = new DateTimeStamp(122);
         DateTimeStamp greater = new DateTimeStamp(124);
         assertEquals(1, greater.compareTo(smaller));
     }
 
     @Test
-    public void testCompareSmallerTimeStampWithoutDateTime() {
+    void testCompareSmallerTimeStampWithoutDateTime() {
         DateTimeStamp smaller = new DateTimeStamp(123);
         DateTimeStamp greater = new DateTimeStamp(124);
         assertEquals(-1, smaller.compareTo(greater));
     }
 
-    /**
```

Comment:
I think you should leave this test in, especially since the compareTo implementation hasn't changed (it should still throw an IllegalStateException). This will help flag possible changes in behavior down the road.